tensorrt_llm/_torch/modules/attention.py

```
@@ -117,6 +119,87 @@ def attn_custom_op_inplace(
+            HelixAllToAllNative.workspace.as_torch_strided_tensor(torch.uint64))
+
+        torch.ops.trtllm.initializeHelixWorkspace(
+            HelixAllToAllNative.workspace_tensor, mapping.cp_rank,
+            mapping.cp_size)
+        torch.cuda.synchronize()
+        HelixCpMnnvlMemory.get_comm(mapping).barrier()
+
+    @staticmethod
+    def alltoall_native(field0: torch.Tensor,
+                        field1: torch.Tensor,
+                        is_fused: bool = False):
```

Ruff (0.14.8)

Unused static method argument: `is_fused`

(ARG004)